NEWS.md

```diff
@@ -0,0 +1,56 @@
+
+# MMAction2 Release Notes
+
+## Master
+
+### Models
+* Add NonLocal module in TSM and I3D [#41](https://github.com/open-mmlab/mmaction2/pull/41)
```

Comment:
`Add NonLocal module in TSM and I3D. ([#41](https://github.com/open-mmlab/mmaction2/pull/41))`